src/common/components/featureSupportLevels/utils.ts

```diff
@@ -47,11 +43,7 @@ export const getLimitedFeatureSupportLevels = (
     }
     return ret;
   } catch (err) {
-    captureException(
-      err,
-      `Failed to get cluster ${cluster.id} feature support levels`,
-      Sentry.Severity.Warning,
-    );
+    console.error(`Failed to get cluster ${cluster.id} feature support levels`);
```

Comment:
Idea: we could pass the `captureException` as an additional arg to this function. Later any consumer can decide "how to capture the exception". In OCM we report it to Sentry in a disconnected environment they might decide to do something else that doesn't require contacting an external service. WDYT?

Reply:
this function is called from a component in common, that's called from another component in common, etc. I think the only way to solve this is with the errorhandler context

Reply:
That's not correct
![image](https://user-images.githubusercontent.com/77008341/151125031-8de4e8d8-2005-4c39-9f64-0be3be8e35ab.png)